hooks/pre-commit

```diff
@@ -2,6 +2,9 @@
 # This file modified from k8s
 # https://github.com/kubernetes/kubernetes/blob/master/hooks/pre-commit
 
```

Comment:
How do you think if we remove this empty line?

Reply:
OK